libgeotiff/cmake/FindGeoTIFF.cmake

```diff
@@ -5,67 +5,100 @@
 # On success, the macro sets the following variables:
 # GEOTIFF_FOUND       = if the library found
 # GEOTIFF_LIBRARIES   = full path to the library
-# GEOTIFF_INCLUDE_DIR = where to find the library headers 
+# GEOTIFF_INCLUDE_DIR = where to find the library headers
 # also defined, but not for general use are
-# GEOTIFF_LIBRARY, where to find the PROJ.4 library.
+# GEOTIFF_LIBRARY
 #
 # Copyright (c) 2009 Mateusz Loskot <mateusz@loskot.net>
+# Copyright (c) 2016 NextGIS <info@nextgis.com>
+# Copyright (C) 2017,2018 Hiroshi Miura
 #
+# Origin from
+# https://svn.osgeo.org/metacrs/geotiff/trunk/libgeotiff/cmake/FindGeoTIFF.cmake
```

Comment:
Do we need this FindGeoTIFF.cmake file at all, to build libgeotiff ? This is more something that should be used by other projects. Or is it just there as a reference for such projects?